images/cluster-proportional-autoscaler/tests/test.yaml

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
The test infra is barfing on this unfortunately. In other tests we have a `cat << EOF > test.yaml` in the test script itself, which isn't great, but it sounds like it would unblock this at least.

Can you make that change to this test so we can merge this? Thanks for writing such a great test! Sorry for the speedbump.